client-resttemplate/swagger.gradle

```diff
@@ -21,7 +24,8 @@ generateSwaggerCode {
 			modelPackage  : "${artifactGroup}.model",
 			apiPackage    : "${artifactGroup}.api",
 			invokerPackage: "${artifactGroup}.client",
-			dateLibrary   : 'java11'
+			dateLibrary   : 'java11',
+			jakarta       : true
```

Comment:
I didn't say this in the last review but it has been nagging me. It might be worth switching from `library = 'resttemplate'` above to `library = 'jersey2'` to be like other service. Then cross service tracing can be [setup like this](https://github.com/DataBiosphere/terra-workspace-manager/blob/3958b8cb975444950305a74a126d59fcc9974136/service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java#L65). Alternatively we could implement a `ClientHttpRequestInterceptor` that does tracing for this kind of client lib.

Reply:
Good shout, I can play with this.

Do you have a sense of what changes would be required to the services interacting with the client library?

Reply:
I think I'd like to work on this separately in its own ticket, so will merge this PR in without it… I couldn't get it to work out of the box but I may be missing something simple.